anxcloud/data_source_vlan_test.go

```diff
@@ -0,0 +1,46 @@
+package anxcloud
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccAnxCloudVLANDataSource(t *testing.T) {
+	// Note: The VLAN for testing can be accessed here: https://engine.anexia-it.com/ipam/prefix/relation/00a239d617504e4ab49122efe0d27657
```

Comment:
I think instead of hard-coding the identifier and name, we should use the VLAN that is already passed in via env vars so that we can change them without updating the code, in case someone deletes them by accident^^

Retrieve the VLANs ID:
```go
	environment.SkipIfNoEnvironment(t)
	vlanID := environment.GetEnvInfo(t).VlanID
```

To resolve VLAN name:
```go
	c := testAccProvider.Meta().(providerContext).legacyClient
	v := vlan.NewAPI(c)
	v.Get ...
```